packages/webapp/src/components/Task/TaskComplete/StepOne.jsx

```diff
@@ -183,7 +183,10 @@ export default function PureCompleteStepOne({
           disabled: !changesRequired,
           isModified: changesRequired,
           locations: selectedTask.locations,
-        })}
+        })
+      ) : (
+        <></>
```

Comment:
null?

Reply:
Thanks 😂 I originally thought I needed to stick some JSX in the map so holdover from that.